### PR TITLE
write a pgsql Stress Test in /test

### PR DESCRIPTION
**Pull request by cdjianghan**

### CODE REVIEW

**Comment by cdjianghan** [comment]
//you can also copy this file to your project and run it or run it in pod on sealos cloud.
//and also you can write a go test to run this Stress Test.
//what you need is to specify a kubeconfig path and your namespace
//example
//func main() {
//	PostgresStressTest("~/.kube/config", "ns-b47ef502")
//}

File: test/testpg/pgstresstest.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+	"log"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"k8s.io/client-go/restmapper"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// 自定义数据
```
Comment (fanux) [inline test/testpg/pgstresstest.go:26]: English

File: test/testpg/pgstresstest.go [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+	}
+	for {
+		ctx := context.Background()
+		createObj, err := PostgresClusterRandomCreate(ctx, dr, obj)
+		if err != nil {
+			return err
+		}
+		time.Sleep(time.Minute * 1)
+		for _, obj := range createObj {
+			err := PostgresGet(ctx, dr, obj)
```
[the remark this answers is not in the document]
Comment (cdjianghan) [reply test/testpg/pgstresstest.go:96]: yes

[the remark this answers is not in the document]
Comment (cdjianghan) [reply test/testpg/pgstresstest.go:96]: get cr status ,running or creating or failed.

File: test/testpg/pgstresstest.go [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+// PostgresGet Get PostgresCluster.
+func PostgresGet(ctx context.Context, dr dynamic.ResourceInterface, obj *unstructured.Unstructured) error {
+	objGET, err := dr.Get(ctx, obj.GetName(), metav1.GetOptions{})
+	if err != nil {
+		log.Printf("select resource ERROR: %v\n", err)
+		return err
+	}
+	fmt.Println(objGET.Object["status"])
```
Comment (fanux) [inline test/testpg/pgstresstest.go:137]: Just check the status, not show it.

Comment (cdjianghan) [reply test/testpg/pgstresstest.go]: eee,what about record it in log? If we go back to check log ,know cluster status is make sense.

**Review by fanux — APPROVED** [review]

**Pull request merged**